[stable/rabbitmq] Update image to new bash container

alemorcuq opened the pull request:
Signed-off-by: Alejandro Moreno <amoreno@bitnami.com>

#### What this PR does / why we need it:
Updates the container to use the new container with bash logic.

#### Special notes for your reviewer:
As the chart has its own logic in a custom command, the changes to the underlying image shouldn't affect the chart. To upgrade, it maybe needed to use the `forceBoot` option, but as far as I know that is also the case with the current version.

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

alemorcuq committed:
Update image to new bash container

Signed-off-by: Alejandro Moreno <amoreno@bitnami.com>

aleeriz committed:
[stable/opa] Added a namespace selector config to admission controller (#14518)

Making the namespace selector configurable allows for more flexibility.
e.g. we want to enable it for only 1 namespace.

Signed-off-by: Ali Rizwan <ari@hellofresh.com>

Commit:
owners: add a new owner (#14530)

Signed-off-by: Hector Fernandez <hfernandez@mesosphere.com>

Commit:
[stable/kong] add Kong Enterprise support (#14509)

* [stable/kong] add Kong Enterprise support

Adds support for Kong Enterprise. The chart now supports either Kong
Enterprise or Kong open source instances, controlled primarily by the
`enterprise.enabled` block of values.yaml. By default, the chart will install
Kong open source.

Signed-off-by: Travis Raines <traines@konghq.com>

* [stable/kong] fix linter errors in values.yaml

Signed-off-by: Travis Raines <traines@konghq.com>

* [stable/kong] fix earlier version bump

Signed-off-by: Travis Raines <traines@konghq.com>

EricLemieux committed:
[incubator/kafka] Adding zookeeper fullnameOverride (#14532)

Signed-off-by: Eric Lemieux <eric@lemieuxdev.com>

oussemos committed:
Add Kubernetes to keywords list (#14523)

Signed-off-by: Oussema CHERNI <oussemos@gmail.com>

samuelb committed:
[stable/kafka-manager] use service.port for addClusters.sh during bootstrap (#14165)

* [stable/kafka-manager] use service.port for addClusters.sh during bootstrap

Signed-off-by: Samuel Barabas <samuel@owee.de>

* Use printf %v instead of convert value to be displayed correctly

Signed-off-by: Samuel Barabas <samuel@owee.de>

dunglas committed:
Revert upgrade hook (#14531)

* Revert "creating secret on pre-upgrade (#14455)"

This reverts commit e55a4c1aefda92654cd2ec5e5c6f9d2851bf1c9c.

Signed-off-by: Kévin Dunglas <dunglas@gmail.com>

* Revert "Create configmap on helm upgrade too (#14447)"

This reverts commit 1c996f98ac8a33ee79a9e163e9478937e0006071.

Signed-off-by: Kévin Dunglas <dunglas@gmail.com>

* [stable/mercure] Revert add upgrade hook

Signed-off-by: Kévin Dunglas <dunglas@gmail.com>

unguiculus committed:
[stable/velero] Remove myself as maintainer (#14537)

Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

Commit:
[stable/helm-exporter] Add support for using ingress (#14442)

* Add support for using ingress

Signed-off-by: Christopher Farrenden <chris.farrenden@domain.com.au>

* Use less convoluted annotation pulling

Signed-off-by: Christopher Farrenden <chris.farrenden@domain.com.au>

* Revert clusterIP conditional

Signed-off-by: Christopher Farrenden <chris.farrenden@domain.com.au>

papanito committed:
[stable/sonarqube] Allow use of custom cacerts (#14413)

* [stable/sonarqube] Allow use of custom cacerts

Signed-off-by: papanito <papanito@wyssmann.com>

* [stable/sonarqube] Bump minor version as this adds a new feature

Signed-off-by: papanito <papanito@wyssmann.com>

* fixup! [stable/sonarqube] Bump minor version as this adds a new feature

Signed-off-by: papanito <papanito@wyssmann.com>

Commit:
stable/drupal: update to 8.7.3 (#14546)

Signed-off-by: Bitnami Containers <containers@bitnami.com>

alemorcuq committed:
Add note about 6.0.0

Signed-off-by: Alejandro Moreno <amoreno@bitnami.com>

carrodher commented on stable/rabbitmq/README.md:
```diff
@@ -208,6 +208,10 @@ $ helm install --set persistence.existingClaim=PVC_NAME rabbitmq
 
+### To 6.0.0
+
+This new version updates the RabbitMQ image to a [new version based on bash instead of JavaScript](https://github.com/bitnami/bitnami-docker-rabbitmq#3715-r18-3715-ol-7-r19). However, since this Chart overwrites the container's command, the changes to the container shouldn't affect the Chart. To upgrade, it may be needed to enable the `fastBoot` option, as it is already the case from upgrading fr ...
```
```suggestion
This new version updates the RabbitMQ image to a [new version based on bash instead of node.js](https://github.com/bitnami/bitnami-docker-rabbitmq#3715-r18-3715-ol-7-r19). However, since this Chart overwrites the container's command, the changes to the container shouldn't affect the Chart. To upgrade, it may be needed to enable the `fastBoot` option, as it is already the case from upgrading from 5.X to 5.Y.  
```
In order to be consistent with previous cases: https://github.com/helm/charts/tree/master/stable/postgresql#400

aerotog committed:
[stable/postgresql] Fix data permission error on pod restart (#14538) (#14540)

The command 'mkdir -p' has no effect if the directory already exists. To ensure the init container sets the mode for an existing data directory, we call 'chmod 700' separately.

Signed-off-by: Tim Guenthner <aerotog@gmail.com>

alemorcuq committed:
Change debug env. variable

Signed-off-by: Alejandro Moreno <amoreno@bitnami.com>

miguelaeh committed:
Add securityContext to MongoDB metrics sidecar container (#14443)

Signed-off-by: Miguel A. Cabrera Minagorri <macabrera@bitnami.com>

alemorcuq committed:
Change wording on 6.0.0 note

Signed-off-by: Alejandro Moreno <amoreno@bitnami.com>

juan131 reviewed (approved):
I'm approving the changes.
@carrodher feel free to process with the merging process when you think it's ready to go

akhfa committed:
[stable/wordpress] Add update strategy (#14415)

* add update strategy for wordpress chart

Signed-off-by: Akhmad Fakhoni Listiyan Dede <akhmadfakhoni@gmail.com>

* fix table formatting

Signed-off-by: Akhmad Fakhoni Listiyan Dede <akhmadfakhoni@gmail.com>

* add update strategy example

Signed-off-by: Akhmad Fakhoni Listiyan Dede <akhmadfakhoni@gmail.com>

* change wordpressUpdateStrategy to updateStrategy

Signed-off-by: Akhmad Fakhoni Listiyan Dede <akhmadfakhoni@gmail.com>

guessi committed:
[stable/nats] Fix ValidationError: unknown field "serviceName" (#14439)

Signed-off-by: guessi <guessi@gmail.com>

denis111 committed:
[stable/rabbitmq] fix Error: enoent with forceBoot on new install (see #13485) (#14491)

* [stable/rabbitmq] fix Error: enoent with forceBoot on new install (see #13485)

Signed-off-by: Denis Kalgushkin <keygenmusic@gmail.com>

* [stable/rabbitmq] Bump chart version for PR 14491 (see #13485)

Signed-off-by: Denis Kalgushkin <keygenmusic@gmail.com>

Constantin07 committed:
[stable/metrics-server] Fix indentation for service labels. (#14541)

Signed-off-by: Constantin Bugneac <cosbug@gmail.com>

madddi committed:
[stable/velero] Make Restic resource requests and limits configurable (#14404)

* [stable/velero] make restic daemonset resource optionally configurable

Signed-off-by: Mathias Merscher <Mathias.Merscher@dg-i.net>

* update README

Signed-off-by: Mathias Merscher <Mathias.Merscher@dg-i.net>

* [stable/velero] bump Chart version

Signed-off-by: Mathias Merscher <Mathias.Merscher@dg-i.net>
